packages/salesforcedx-vscode-core/src/decorators/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export { showDemoMode } from './demo-mode-decorator';
```

Comment:
@Sweetman - I organized all the different decorators under the new decorators folder.